test/d-b-n_smoke/arrays.html

```diff
@@ -296,6 +328,16 @@
                         })]);
 
                     });
+                    it('when adding element to array patch should be sent', function (done) {
+                        ppclient.addEventListener('patchsent', patchSentEventCallback);
+                        puppet.ignoreAdd = false;
+                        domBind.splice('model.people_push', 1, 0, {first: 'Abraham'});
```

Comment:
I agree with the latter, especially since the whole onLocalChange affair is not rock-solid (see puppetjs/puppet-polymer-client#26)